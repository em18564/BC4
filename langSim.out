Running on host compute147.bc4.acrc.priv
Time is Fri Feb 17 10:47:31 GMT 2023
Directory is /user/home/em18564/LanguageSim
Slurm job ID is 11204725
This job runs on the following machines:
compute147
Running on host compute121.bc4.acrc.priv
Time is Fri Feb 17 10:47:31 GMT 2023
Directory is /user/home/em18564/LanguageSim
Slurm job ID is 11204820
This job runs on the following machines:
compute121
