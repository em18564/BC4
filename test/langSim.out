Running on host compute321.bc4.acrc.priv
Time is Thu Feb 16 15:16:37 GMT 2023
Directory is /user/home/em18564/LanguageSim/test
Slurm job ID is 11192889
This job runs on the following machines:
compute321
Running on host compute520.bc4.acrc.priv
Time is Thu Feb 16 15:16:37 GMT 2023
Directory is /user/home/em18564/LanguageSim/test
Slurm job ID is 11192890
This job runs on the following machines:
compute520
slurmstepd: error: *** JOB 11192889 ON compute321 CANCELLED AT 2023-02-16T15:19:39 ***
slurmstepd: error: *** JOB 11192841 ON compute177 CANCELLED AT 2023-02-16T15:19:39 ***
slurmstepd: error: *** JOB 11192842 ON compute261 CANCELLED AT 2023-02-16T15:19:39 ***
slurmstepd: error: *** JOB 11192843 ON compute373 CANCELLED AT 2023-02-16T15:19:39 ***
slurmstepd: error: *** JOB 11192840 ON compute107 CANCELLED AT 2023-02-16T15:19:39 ***
slurmstepd: error: *** JOB 11192890 ON compute520 CANCELLED AT 2023-02-16T15:19:39 ***
